api-client/src/endpoints/createDisclaimerEndpoint.ts

```diff
@@ -25,7 +25,7 @@ export default (baseUrl: string): Endpoint<ParamsType, PageModel> =>
         path: json.path,
         title: json.title,
         content: json.content,
-        lastUpdate: moment.tz(json.modified_gmt, 'GMT'),
+        lastUpdate: DateTime.fromJSDate(new Date(json.modified_gmt), { zone: 'GMT' }),
```

Comment:
@steffenkleinle your suggestion here as well returns `DateInvalid` string

Reply:
Sorry, I had a typo. It is `last_updated`, mind the additional `d` in  the end. So it should be
```suggestion
        lastUpdate: DateTime.fromISO(json.last_updated),
```

Reply:
ahh I haven't noticed a typo there. I just copy and pasted. I will try to replace again:)

Reply:
@steffenkleinle but why we are changing from `(json.modified_gmt)` to `(json.last_updated)`.

Reply:
Because the standard ISO 8601 should be used for sharing dates/times because everything else isn't a standard/error prone. `modified_gmt` uses, as the name says, gmt time and not an ISO_8601 date format. Therefore we wanted to change to `last_updated` which is ISO 8601 for a long time.

Reply:
@steffenkleinle when I replace it to `toISO()` it returns slightly changed values and does not pass testing as well as in another endpoints